lib/tasks/retire.rake

```diff
@@ -7,8 +7,9 @@ namespace :retire do
 
     content_item_publisher = ContentItemPublisher.new
 
-    content_item_publisher.unpublish(args.content_id)
-    content_item_publisher.publish_redirect(args.base_path, args.destination)
+    content_item_publisher.unpublish_with_redirects(
+      args.content_id, args.path, args.destination
```

Comment:
Should `args.path` be `args.base_path`?

Reply:
Yep, good spot, I've updated the commit.